src/components/GameTabs.vue

```diff
@@ -263,7 +264,7 @@ const gameIndex = computed(() =>
   }),
 );
 
-const displayGameIndex = computed(() => (gameIndex.value === -1 ? 0 : gameIndex.value));
+displayGameIndex.value = computed(() => (gameIndex.value === -1 ? 0 : gameIndex.value));
```

Comment:
There appears to be an issue with how `displayGameIndex` is being handled in these changes as it's first assigned to be a ref, and then reassigned to a computed property. 

Instead, it should be possible to just declare it as a computed value:
```suggestion
const displayGameIndex = computed(() => (gameIndex.value === -1 ? 0 : gameIndex.value));
```